### DISCUSSION TITLE
Erreur identifiants multiples

### DISCUSSION THREAD

**Discussion by mxtla** [danymat/INSAnonym-utils#64] [Q&A, Bug report & Support] [answered]
Bonjour, 

Je relance cette discussion car dans un test annexe, j'ai essayé de changer les identifiants de chaque individu par semaine, mais mon test semble également refusé. Est-ce un problème sur mon fichier ?
#56 
PrivUQAM

PROBLEME RESOLU

Merci d'avance, 

Margaux TELA

**Comment by danymat** [comment, marked as answer]
Je lock cette conversation, car le problème est résolu par l'auteur
